back/app/services/xlsx_export/value_visitor.rb

```diff
@@ -79,6 +79,14 @@ def visit_page(field)
       # The field does not capture data, so there is no value.
     end
 
+    def visit_file_upload(field)
+      file_id = value_for(field)
+      return unless file_id
+
+      idea_file = model.idea_files.detect { |file| file.id == file_id }
```

Comment:
If we don't do anything about the template issue, I expect that some XLSX exports will no longer work.

Reply:
@sebastienhoorens What is the relationship between templates and XLSX exports?